src/main/java/com/ldtteam/structurize/placement/structure/IStructureHandler.java

```diff
@@ -252,5 +252,14 @@ default BlockPos getProgressPosInWorld(final BlockPos localPos)
      * @param worldPos the world pos.
      * @return the right block (classically biome dependent).
      */
+    @Deprecated(forRemoval = true, since = "1.18.2")
     BlockState getSolidBlockForPos(BlockPos worldPos);
```

Comment:
I would keep this, so that we can still have "a block we want" and not "only worldgen block"

Reply:
name addressed